Tax Rate information update (#309)
codatio/codat-docs: docs/sync-for-expenses/configandcategorize.md

The tax rates enable your SMB's to accurately track taxes against purchases and, depending on the locale, allow them to recoup the tax.

> Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example: Applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%. 

Reviewer: ```suggestion
Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example, applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%. 
```

> Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example, applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%. 

Reviewer 2: ```suggestion
Accounting systems typically store a set of taxes and associated rates within the accounting package. This means users don't have to look up or remember the rate for each type of tax. For example, applying the tax "UK sales VAT" to the line items of an invoice adds the correct tax rate of 20%. 
```

> Assigning a tax rate to a transaction is mandatory unless the transaction is a [transfer](/sync-for-expenses/sync-process/expense-transactions#transaction-types). In some cases, your customers might not need to track tax on expenses. We recommend assigning a default tax code for 0% from the accounting package for these transactions.

Reviewer 2: ```suggestion
Assigning a tax rate to a transaction is mandatory unless the transaction is a transferIn or transferOut (see [Transaction types](/sync-for-expenses/sync-process/expense-transactions#transaction-types). In some cases, your customers might not need to track tax on expenses. We recommend assigning a default tax code for 0% from the accounting package for those transactions.
```

Reviewer 2: I don't think the link text of "transfer" is clear enough. I thought it was going to direct me to the data model.

Reviewer 2: Use "those" to refers to things which have already been mentioned in the sentence :-) [1]

[1] https://www.collinsdictionary.com/dictionary/english/those
